### PR TITLE
Update contribution guidelines

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://dev.azure.com/blackbaud/Products/_boards/board/t/SKY%20UX%20Components/Stories/?workitem=1602600.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-contribution-updates](https://developer.blackbaud.com/skyux?_sp.spa=pr-contribution-updates)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-contribution-updates](https://developer.blackbaud.com/skyux?_sp.spa=pr-contribution-updates)

**Review by blackbaud-paulah — APPROVED** [review]

**Review by Blackbaud-MichaelCollier — APPROVED** [review]
Looks great, thanks, John! We'll modify based on user feedback but this is definitely what we need to get started.

File: src/app/contribute/contribution-process/file-issue/index.html [hunk 1]
```diff
@@ -24,4 +29,37 @@
...
+    <li>
+      <p>
+        Create a GitHub issue in the <a stacheRouterLink="/contribute/packages">applicable repo</a>.
+      </p>
+      <sky-restricted-view>
+        <sky-alert alertType="info">
+          Blackbaud developers should report bugs in ADO instead of GitHub. The SKY UX team uses ADO dashboards to manage work, but other than the location for filing issues, the process remains the same, including the availability of a <a href="https://dev.azure.com/blackbaud/Products/_workitems/create/Bug?templateId=762f6a46-de2a-49ee-b2c4-c3bb0ef60de0&ownerId=26bb0bf6-45ac-482d-b33e-02e252c297e3">template for filing bugs</a>.
+        </sky-alert>
+      </sky-restricted-view>
+      <sky-restricted-content-alert></sky-restricted-content-alert>
```
Comment (Blackbaud-TrevorBurch) [inline src/app/contribute/contribution-process/file-issue/index.html:49]: Not a huge deal - but wanted to ask. We currently have two alerts in two different places on this page. Is that what we want or do we just want to place one alert at the top of the page about content existing for BB employees?

File: src/app/contribute/contribution-process/index.html [hunk 2]
```diff
@@ -90,6 +96,12 @@
...
         </p>
+        <sky-restricted-view>
+          <sky-alert alertType="info">
+            Blackbaud developers should file documentation issues in ADO instead of GitHub. The SKY UX team uses ADO dashboards to manage work, but other than the location for filing issues, the process remains the same, including the availability of a <a href="https://blackbaud.visualstudio.com/Products/_workitems/create/Feature?templateId=553e8cbd-12ff-4e72-a44f-e47e2de5f0a8&ownerId=26bb0bf6-45ac-482d-b33e-02e252c297e3">template for filing issues</a>.
+          </sky-alert>
+        </sky-restricted-view>
+        <sky-restricted-content-alert></sky-restricted-content-alert>
```
Comment (Blackbaud-TrevorBurch) [inline src/app/contribute/contribution-process/index.html:104]: Same question on this page about 1 alert or two?

**Review by Blackbaud-TrevorBurch — COMMENTED** [review]
@blackbaud-johnly - one quick question in two places. Other than that - LGTM

File: src/app/contribute/contribution-process/file-issue/index.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/contribute/contribution-process/file-issue/index.html]: I removed the second alert on this topic.

File: src/app/contribute/contribution-process/index.html (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/contribute/contribution-process/index.html:104]: As discussed, we'll go with 2 alerts here.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-contribution-updates](https://developer.blackbaud.com/skyux?_sp.spa=pr-contribution-updates)

**Pull request merged**